tools/pages/permit-holders/filter-permit-holders.ts

```diff
@@ -0,0 +1,38 @@
+import { gql } from '@apollo/client'; // gql tag
+import { ApplicantsFilter, Applicant } from '@lib/graphql/types';
+
+export const FILTER_PERMIT_HOLDERS_QUERY = gql`
+  query FilterPermitHoldersQuery($filter: ApplicantsFilter) {
+    applicants(filter: $filter) {
+      result {
+        firstName
+        middleName
+        lastName
+        dateOfBirth
+        addressLine1
+        city
+        postalCode
+        email
+        phone
+        recentPermit {
+          expiryDate
+          rcdPermitId
+        }
+        status
+        id
+      }
+      totalCount
+    }
+  }
+`;
+
+export type FilterPermitHoldersRequest = {
+  filter: ApplicantsFilter;
+};
+
+export type FilterPermitHoldersResponse = {
+  applicants: {
+    result: [Applicant];
```

Comment:
I think this will be an array of one single `Applicant` lol

We want to use TypeScript's `Pick` so that we can get type checking and make sure that we are exactly receiving the fields we want. So it would be something like:

```suggestion
    result: ReadonlyArray<
      Pick<
        Applicant,
        | 'firstName'
        | 'middleName'
        | 'lastName'
        ...
      >
    >;
```